.github/workflows/release-extension.yml

```diff
@@ -42,6 +42,14 @@ jobs:
           release_name: ${{ steps.derive_tag.outputs.TAG }}
           draft: false
           prerelease: false
+      - name: Upload Firefox extension to mozilla addon
+        uses: DarkFlorist/firefox-addon@v1-body-size-increase
+        with:
+          uuid: '{3c236fbc-9114-43ed-a224-0cd1834aec4d}'
```

Comment:
This is our extensions ID on Firefox store?

Reply:
Yes.